lib/audited/auditor.rb

```diff
@@ -43,12 +43,14 @@ def audited(options = {})
 
         class_attribute :audit_associated_with,   instance_writer: false
         class_attribute :audited_options,       instance_writer: false
+        class_attribute :keep_audits,             instance_writer: false
```

Comment:
Perhaps `max_audits` is more descriptive of what this setting is? Maybe also ensure it is a positive integer in the option normalization?